pkg/controller/admissionchecks/multikueue/testdata/kubeconfig

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```

Comment:
This looks like testing fake kubeconfig. We generate now them on the fly and record in tmp dir. Check the code around https://github.com/kubernetes-sigs/kueue/blob/ff15f19c47cb6a709fdd982a3ff6eceae50dbc5c/pkg/controller/admissionchecks/multikueue/multikueuecluster_test.go#L109C6-L115

cc @mszadkow

Reply:
Thanks for the link! I've removed the fake kubeconfig.